easybuild/easyconfigs/m/muParser/muParser-2.2.5-GCCcore-10.2.0.eb

```diff
@@ -0,0 +1,36 @@
+easyblock = 'ConfigureMake'
+
+name = 'muParser'
+version = '2.2.5'
+
+homepage = 'https://beltoforion.de/article.php?a=muparser'
+
+description = """
+ muParser is an extensible high performance math expression parser library
+ written in C++. It works by transforming a mathematical expression into
+ bytecode and precalculating constant parts of the expression.
+"""
+
+toolchain = {'name': 'GCCcore', 'version': '10.2.0'}
+toolchainopts = {'pic': True}
+
+source_urls = ['https://github.com/beltoforion/muparser/archive/']
+sources = ['v%(version)s.tar.gz']
+checksums = ['0666ef55da72c3e356ca85b6a0084d56b05dd740c3c21d26d372085aa2c6e708']
+
+builddependencies = [
+    ('binutils', '2.35'),
+]
+
+dependencies = [
+    ('CMake', '3.18.4'),
```

Comment:
Does this really need `CMake`, since it's using `ConfigureMake` as easyblock?

If so, please add a comment in the easyconfig to clarify...